pkg/service/agent_test.go

```diff
@@ -26,6 +27,13 @@ func TestAgentServiceNameAndPorts(t *testing.T) {
 
 	for _, port := range svc.Spec.Ports {
 		ports[port.Port] = true
+		switch port.Port {
+		case
+			5775,
+			6831,
+			6832:
+			assert.Equal(t, port.Protocol, corev1.ProtocolUDP, "Expected port %v to be UDP, but wasn't", port)
```

Comment:
The second argument to `Equal` is the expected value, followed by the actual value:

```
assert.Equal(t, corev1.ProtocolUDP, port.Protocol, ...)
```

Of course, this is not blocking the PR :)

Reply:
Well, right! But then, for the same reason, the next assert arguments should also be swapped as well 😉 

After rebasing and seeing the test cases fail, I ran into the same failures locally. 
And they do, indeed, complain ports or port forwarding. So I'm not sure did maybe sth change from 1.14 to 1.15? In that case, it is good that the test cases fail.
```
E1118 15:55:18.902332   45722 portforward.go:400] an error occurred forwarding 57500 -> 14268: error forwarding port 14268 to pod 65359bd8523f97b65fb2350d0639d7bc5de8e68e7c752614c2c63e61d76755d0, uid : exit status 1: 2019/11/18 14:55:18 socat[24686] E connect(5, AF=2 127.0.0.1:14268, 16): Connection refused
E1118 15:57:18.950977   45722 portforward.go:408] error closing listener: close tcp4 127.0.0.1:57499: use of closed network connection
E1118 15:57:18.951003   45722 portforward.go:408] error closing listener: close tcp6 [::1]:57499: use of closed network connection
E1118 15:57:18.952080   45722 portforward.go:408] error closing listener: close tcp4 127.0.0.1:57500: use of closed network connection
E1118 15:57:18.953160   45722 portforward.go:408] error closing listener: close tcp6 [::1]:57500: use of closed network connection
```
To my understanding, these errors occur in https://github.com/jaegertracing/jaeger-operator/blob/3020ae448e003c72b7cc6d16144e68dae9305415/test/e2e/smoketest.go#L69 but are actually originating from the method https://github.com/jaegertracing/jaeger-client-go/blob/ab94682f276c8c307e7fcd3f551c48ad459eab8f/tracer.go#L366-L377 not checking are the resources closable at all. So the actual error seems to be an upstream issue.